ando/tools/generator/AnDOGenerator.py

```diff
@@ -116,8 +118,10 @@ def basedir(self):
     @basedir.setter
     def basedir(self, basedir):
         """
-        Args:
-            basedir (str,path): path to the projects base folder (project root)
+        Parameters
+        ----------
+            basedir : (str,path)
```

Comment:
You are inconsistent with the indentation, so do the variable names need to be aligned with `Parameters` or indented by one level?

Reply:
It's no indentation sorry. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy